src/components/common/inside/headers/blueHeader.tsx

```diff
@@ -0,0 +1,16 @@
+import 'components/common/inside/headers/blueHeader.css'
+
+import rightArrow from 'assets/images/arrow_left.png'
+
+const BlueHeader = ({text}: {text: string}) => {
```

Comment:
It's a good practice to do the following in case you'll need to add more props to this component:
```suggestion
interface BlueHeaderProps {
  text: string;
}
const BlueHeader = ({text}: BlueHeader) => {
```